src/data-root.ts

```diff
@@ -85,9 +85,6 @@ export async function update(
   if (fetchController) fetchController.abort()
   fetchController = new AbortController()
 
-  // Ensure peer connection
-  await ipfs.reconnect()
```

Comment:
this is no longer needed?

Reply:
Is handled in https://github.com/fission-suite/auth-lobby/blob/master/src/Javascript/Worker.js#L61 now.